### ISSUE TITLE
error running serve.js

### ISSUE THREAD

**Issue by kurtinlane** [LDSorg/frontend-oauth2-angular-example#1]
just followed the readme and got stuck on this error 

<img width="511" alt="screenshot 2015-07-12 15 49 40" src="https://cloud.githubusercontent.com/assets/5891949/8640456/2f0c0ec6-28ae-11e5-9215-993cb634f2e6.png">

**Comment by coolaj86** [comment]
Yeah... something wonky has happened with one of the APIs I was using and the certificates are getting a false negative on validation... strange because Firefox and Safari and Chrome all like the certificate just fine.

Anyway, I have the APIs down currently while I go through some red tape process with ICS (the IT dept of the Corp of Pres Bishopric)

**Comment by coolaj86** [comment]
I did fix the problem related to the original issue, but I won't be able to work an alternative API until around September.
